### PR TITLE
Add Vesting contracts + Centrifuge RWAs + Set changelog version to 1.9.5 + other minor stuff

### PR DESCRIPTION
**Pull request by gbalabasquer**
# Description

# Contribution Checklist

- [ ] PR title starts with `(PE-<TICKET_NUMBER>)`
- [ ] Code approved
- [ ] Tests approved
- [ ] CI Tests pass

# Checklist

- [ ] Every contract variable/method declared as public/external private/internal
- [ ] Consider if this PR needs the `officeHours` modifier
- [ ] Verify expiration (`4 days + 2 hours` monthly and `30 days` for the rest)
- [ ] Verify hash in the description matches [here](https://emn178.github.io/online-tools/keccak_256.html)
- [ ] Validate all addresses used are in Goerli changelog or known
- [ ] Notify any external teams affected by the spell so they have the opportunity to review
- [ ] Deploy spell to Goerli `ETH_GAS="XXX" ETH_GAS_PRICE="YYY" make deploy`
- [ ] Ensure contract is verified on `Goerli` etherscan
- [ ] Change test to use Goerli spell address and deploy timestamp
- [ ] Keep `Goerli-DssSpell.sol` and `Goerli-DssSpell.t.sol` the same, but make a copy in `archive`
- [ ] `squash and merge` this PR

### CODE REVIEW

File: src/Goerli-DssSpell.sol [hunk 1]
```diff
@@ -44,110 +50,313 @@ interface RwaOutputConduitLike {
...
+            urnID: "RWA006_A_URN",
+            inputConduitID: "RWA006_A_INPUT_CONDUIT",
+            outputConduitID: "RWA006_A_OUTPUT_CONDUIT",
+            pipID: "PIP_RWA006",
+            ilk: "RWA006-A",
+            ilk_string: "RWA006",
+            ilkRegistryName: "RWA006-A: Centrifuge: Alternative Equity Advisers",
+            RATE: TWO_PCT,
+            CEIL: 0 * MILLION,
+            PRICE: 0 * WAD,
```
Comment (godsflaw) [inline src/Goerli-DssSpell.sol:231]: In `kovan` we gave this a debt ceiling for testing:
```suggestion
            CEIL: 20 * MILLION,
            PRICE: 20_808_000 * WAD,
```

File: src/Goerli-DssSpell.t.sol [hunk 2]
```diff
@@ -1321,6 +1321,151 @@ contract DssSpellTest is DSTest, DSMath {
...
+            calc_tau:     0,
+            calc_step:    0,
+            calc_cut:     0
+        });
+        afterSpell.collaterals["RWA006-A"] = CollateralValues({
+            aL_enabled:   false,
+            aL_line:      0 * MILLION,
+            aL_gap:       0 * MILLION,
+            aL_ttl:       0,
+            line:         0 * MILLION,
```
Comment (godsflaw) [inline src/Goerli-DssSpell.t.sol:1445]:
```suggestion
            line:         20 * MILLION,
```

**Review by godsflaw — COMMENTED** [review]
```haskell
Running 13 tests for src/Goerli-DssSpell.t.sol:DssSpellTest
[PASS] testOnTime() (gas: 10039381)
[PASS] testVestDAI() (gas: 10202483)
[PASS] testFailTooEarly() (gas: 3745)
[PASS] testSpellIsCast_GENERAL() (gas: 19312316)
[PASS] testFailWrongDay() (gas: 3833)
[PASS] testCastCost() (gas: 10041132)
[PASS] testVestMKR() (gas: 10101547)
[PASS] testRWAManagerWards() (gas: 48230)
[PASS] testNewChainlogValues() (gas: 10096602)
[PASS] test_RWA_values() (gas: 12098045)
[PASS] testNewIlkRegistryValues() (gas: 10141532)
[PASS] testFailTooLate() (gas: 4075)
[PASS] testRwaDrawUpToDebtCeiling() (gas: 553756)
```

File: src/Goerli-DssSpell.t.sol [hunk 3]
```diff
@@ -1321,6 +1321,151 @@ contract DssSpellTest is DSTest, DSMath {
...
+            flip_beg:     0,
+            flip_ttl:     0,
+            flip_tau:     0,
+            flipper_mom:  0,
+            dog_hole:     0,
+            clip_buf:     0,
+            clip_tail:    0,
+            clip_cusp:    0,
+            clip_chip:    0,
+            clip_tip:     0,
```
[the remark this answers is not in the document]
Comment (gbalabasquer) [reply src/Goerli-DssSpell.t.sol:1346]: As agreed I've set the others to 0.

**Review by godsflaw — APPROVED** [review]
```haskell
Running 13 tests for src/Goerli-DssSpell.t.sol:DssSpellTest
[PASS] testOnTime() (gas: 10099436)
[PASS] testVestDAI() (gas: 10262538)
[PASS] testFailTooEarly() (gas: 3745)
[PASS] testSpellIsCast_GENERAL() (gas: 19372980)
[PASS] testFailWrongDay() (gas: 3833)
[PASS] testCastCost() (gas: 10101187)
[PASS] testVestMKR() (gas: 10161602)
[PASS] testRWAManagerWards() (gas: 48230)
[PASS] testNewChainlogValues() (gas: 10156657)
[PASS] test_RWA_values() (gas: 12238058)
[PASS] testNewIlkRegistryValues() (gas: 10201587)
[PASS] testFailTooLate() (gas: 4075)
[PASS] testRwaDrawUpToDebtCeiling() (gas: 553401)
```

**Review by godsflaw — APPROVED** [review]
```haskell
Running 13 tests for src/Goerli-DssSpell.t.sol:DssSpellTest
[PASS] testOnTime() (gas: 9859155)
[PASS] testVestDAI() (gas: 10022257)
[PASS] testFailTooEarly() (gas: 8745)
[PASS] testSpellIsCast_GENERAL() (gas: 23513773)
[PASS] testFailWrongDay() (gas: 8833)
[PASS] testCastCost() (gas: 9860906)
[PASS] testVestMKR() (gas: 9925821)
[PASS] testRWAManagerWards() (gas: 48230)
[PASS] testNewChainlogValues() (gas: 9916376)
[PASS] test_RWA_values() (gas: 11997768)
[PASS] testNewIlkRegistryValues() (gas: 9961290)
[PASS] testFailTooLate() (gas: 9075)
[PASS] testRwaDrawUpToDebtCeiling() (gas: 556016)
```

**Pull request merged**